docs/build/building-android-applications/android-signing-for-google-play.md

```diff
@@ -35,7 +35,7 @@ In order to keep your config in Appcircle, you need to Navigate through:
 
 Alternatively, you can accomplish the same within environment variables. The environment variable for this action is `AC_V2_SIGN` . More information is found at the document below:
 
-<ContentRef url="../environment-variables/why-to-use-environment-variables-and-secrets">
+<ContentRef url="../environment-variables">
```

Comment:
@furkanarabaci "ContentRef" is broken here. I'm fixing... :wrench: 

![Screenshot from 2023-11-30 14-39-42](https://github.com/appcircleio/appcircle-docusaurus/assets/7355362/51d12e3d-db20-421d-ab32-656ba7e64878)